Taskfile.yml

```diff
@@ -117,6 +119,43 @@ tasks:
         # stderr for logging.
         sh: echo '{{.INPUT}}' | ./bin/check 2>&1 1>/dev/null
 
+  test:acceptance:
```

Comment:
Maybe you can add a `precondition` step to automatically run `fly login` when we need to run acceptance tests (and maybe also a defer to fly logout? )

Reply:
Actually at the beginning I had `fly login` as the first step of target `test:acceptance`. Then I found it annoying, because it keeps popping up the web browser also if it is already logged in.

Currently you are supposed to call once `task test:acceptance:fly-login` and then call `task test:acceptance` as many times as you need. I found this works well when developing the acceptance tests or a new feature. I think it is the less-worse trade-off...